src/main/java/net/dv8tion/jda/core/entities/Game.java

```diff
@@ -161,7 +161,7 @@ public static Game of(String name, String url)
      */
     public static boolean isValidStreamingUrl(String url)
     {
-        return url != null && url.matches("^https?://(www\\.)?twitch\\.tv/.+");
```

Comment:
Why was the `^` removed?

Reply:
It doesn't matter when you use `matches`